.github/workflows/npm-publish.yml

```
@@ -0,0 +1,18 @@
+name: Publish to NPM
+on:
+  # When a release is created on GitHub, publish to NPM
+  release:
+    types: [created]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
```

We might want to use node 18

```suggestion
          node-version: 18
```